modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -120,6 +120,39 @@ definition {
 		}
 	}
 
+	@description = "LPS-161923 - Verify that it's possible to search for notifications"
+	@priority = "3"
+	test CanSearchForNotifications {
+		task ("Go to Templates and add Notification Templates") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			for (var templateName : list "Test,Liferay,Site") {
+				ObjectNotifications.addNewNotificationTemplate(
+					emailBody = "Email Body",
+					entryTo = "test@liferay.com",
+					fromAddress = "Liferay",
+					fromName = "Test Test",
+					name = "Notification ${templateName}",
+					subject = "Subject");
+
+				Button.clickSave();
+			}
+		}
+
+		task ("Search for Template by name") {
+			Refresh();
+
+			ObjectNotifications.searchNotificationTemplate(entryName = "Liferay");
+		}
+
+		task ("Assert that Notification Template is present") {
+			ObjectNotifications.viewNotificationTemplate(notificationTemplateName = "Notification Liferay");
+		}
```

Comment:
Assert only Notification Liferay is present

We also want to assert that the other Notifications aren't present as well, since this test will still pass if it returns all of the entries.